app/assets/javascripts/jquery.live-lookup.js

```diff
@@ -17,18 +17,29 @@
       $.getJSON(live_lookup_url, function(data){
           for (var i=0, length=data.length; i < length; i++) {
             var live_data = data[i];
-            var dom_item = $("[data-barcode='" + live_data.barcode + "']");
+
+            if (live_data.service == 'FOLIO') {
```

Comment:
Maybe I should have asked on #3180, but I'm wondering if we want to reconcile the two APIs so we don't need to do this? If Sirsi returned item_id, status, and is_available fields we wouldn't have to care?

Reply:
I struggled with what to do about this. I opted to align the `LiveStatus::Folio` response more with Folio's view of the world, thinking it will be less confusing in the long run if we describe the response accurately. I think we're going to need to be clear about what's a location and what's a status with Folio and this was an attempt to try to do that.

Reply:
Sorry, I misread. I'll see if changing the Sirsi response to look more like Folio would break anything.